<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ASUPSDlg</class>
 <widget class="QDialog" name="ASUPSDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1341</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AS UPS 1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/res/ups.png</normaloff>:/res/ups.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frHeader">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>51</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbUPSURL">
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>UPS:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leURL">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>ups@192.168.100.71</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbQuery">
        <property name="text">
         <string>Query</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbQueryPeriodically">
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Auto query</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbFrom">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtFrom">
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="displayFormat">
         <string>yyyy.MM.dd HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTo">
        <property name="text">
         <string>To:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateTimeEdit" name="dtTo">
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="displayFormat">
         <string>yyyy.MM.dd HH:mm</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbUpdate">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbLast">
        <property name="text">
         <string>Last:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbLast">
        <item>
         <property name="text">
          <string>5 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30 min</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 hour</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 day</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 month</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1 year</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLastUpdate">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>281</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbUpdateDB">
        <property name="text">
         <string>Update DB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbAutoUpdateDB">
        <property name="text">
         <string>Auto</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frfData">
       <property name="maximumSize">
        <size>
         <width>405</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTableWidget" name="twData">
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>210</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Item</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="frGraph" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
